### PR TITLE
adding AppCreationScripts

### PR DESCRIPTION
**Pull request by derisen**
- Added apps.json for portal/quickstart
- Added tokenization for for sovereign clouds and graph endpoints.
- Added powershell scripts for portal/registration

### CODE REVIEW

File: AppCreationScripts/AppCreationScripts.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+1. In PowerShell run:
+   ```PowerShell
+   Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+   ```
+1. Run the script to create your Azure AD application and configure the code of the sample application accordingly. (Other ways of running the scripts are described below)
+   ```PowerShell
+   cd .\AppCreationScripts\
+   .\Configure.ps1
+   ```
+1. Open the Visual Studio solution and click start
```
Comment (jasonnutter) [inline AppCreationScripts/AppCreationScripts.md]: General comment: Add relevant punctuation at the end of each bullet/item.

e.g.

1. Open the Visul Studio solution and click start:
1. Run the script to create your Azure AD application and configure the code of the sample application accordingly (Other ways of running the scripts are described below):

File: AppCreationScripts/AppCreationScripts.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+. .\Cleanup.ps1 -Credential $mycreds
+. .\Configure.ps1 -Credential $mycreds
+```
+
+Of course, in real life, you might already get the password as a `SecureString`. You might also want to get the password from KeyVault.
+
+#### Option 3 (Interactive, but create apps in a specified tenant)
+
+  if you want to create the apps in a particular tenant, you can use the following option:
+- open the [Azure portal](https://portal.azure.com)
```
Comment (jasonnutter) [inline AppCreationScripts/AppCreationScripts.md]: Capitalize "If" and "Open"

File: AppCreationScripts/AppCreationScripts.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+5. Go to the `AppCreationScripts` sub-folder. From the folder where you cloned the repo,
+    ```PowerShell
+    cd AppCreationScripts
+    ```
+6. Run the scripts. See below for the [four options](#four-ways-to-run-the-script) to do that.
+7. Open the Visual Studio solution, and in the solution's context menu, choose **Set Startup Projects**.
+8. select **Start** for the projects
+
+You're done. this just works!
```
Comment (jasonnutter) [inline AppCreationScripts/AppCreationScripts.md]: "This just works!"

File: AppCreationScripts/AppCreationScripts.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+### Four ways to run the script
+
+We advise four ways of running the script:
+
+- Interactive: you will be prompted for credentials, and the scripts decide in which tenant to create the objects,
```
Comment (jasonnutter) [inline AppCreationScripts/AppCreationScripts.md]: Periods instead of commas at the end.

File: AppCreationScripts/AppCreationScripts.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+- non-interactive: you will provide credentials, and the scripts decide in which tenant to create the objects,
+- Interactive in specific tenant:  you will provide the tenant in which you want to create the objects and then you will be prompted for credentials, and the scripts will create the objects,
+- non-interactive in specific tenant: you will provide tenant in which you want to create the objects and credentials, and the scripts will create the objects.
+
+Here are the details on how to do this.
```
Comment (jasonnutter) [inline AppCreationScripts/AppCreationScripts.md]: Not sure this sentence is needed.

**Review by jasonnutter — CHANGES_REQUESTED** [review]
A few comments re: grammer. Also, I wonder if screenshots would be helpful? Great job otherwise!

File: AppCreationScripts/AppCreationScripts.md (hunk 1 above)
Comment (derisen) [reply AppCreationScripts/AppCreationScripts.md]: My apologies, I haven't read this through at all -will do.

**Review by mstada — APPROVED** [review]
Portal integration points LGTM.

**Comment by derisen** [comment]
Grammar issues fixed. Hope it's better now

**Pull request merged**